keps/sig-scheduling/20200114-taint-based-evictions.md

```diff
@@ -0,0 +1,101 @@
+---
+title: Promote Taint Based Evictions to GA
+authors:
+  - "@damemi"
+owning-sig: sig-scheduling
```

Comment:
sig-scheduling is not the owning sig of this feature, the scheduler doesn't include any code specifically for this feature.

Reply:
@liggitt which sig does the node lifecycle controller falls under?

Reply:
would that be sig-node?

Reply:
probably sig-node

Reply:
or sig-cloud-provider for the "not reachable nodes" aspect